symbols/arm9.yml

```diff
@@ -7799,6 +7799,25 @@ arm9:
         
         r0: team member index
         return: roster index if the team member is active, -1 otherwise
+    - name: TryAddRosterMonsterToTeam
+      address:
+        EU: 0x2056AD0
+        NA: 0x2056754
+      description: |-
+        Attempts to add a monster from the roster to the active team.
```

Comment:
I'll change it. Although "team member table" doesn't sound too clear to me. Maybe it could be changed (globally) to "team assembly" or something like that?

Reply:
Well `struct team_member_table` holds information about the team and its various arrangements, including all the members in the assembly, the different rosters, and the currently active roster. So it's not just the assembly.

Reply:
I see. I guess it's alright then.